lib/ramble/docs/tutorials/hello_world.rst

```diff
@@ -0,0 +1,219 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _hello_world_tutorial:
+
+=====================================================
+1) Getting Started Running A "Hello World" Experiment
+=====================================================
+
+This tutorial will provide a basic introduction to navigating Ramble and running
+experiments. In this tutorial, you will set up and run a basic experiment using
+``hostname``.
+
+Installation
+============
+
+To install Ramble, see the :doc:`../getting_started` guide.
+
+**NOTE**: This tutorial does not require ``spack`` to be installed or configured.
+
+Ramble Basics
+=============
+
+Before beginning, we will learn some basics about ``ramble`` and its available
+commands.
+
+Available Applications
+----------------------
+
+Once Ramble is installed, viewing the available application definitions is as
+simple as executing:
+
+.. code-block:: console
+
+    $ ramble list
+
+The ``ramble list`` command also takes a query string, which can be used to
+filter available application definitions. For example:
+
+.. code-block:: console
+
+    $ ramble list h*
```

Comment:
Does this definitely work? Does not work for me on OSX with either zsh or bash (but dropping the * does show any package with an h in, just not starting with)

Reply:
Good catch. It's the same on my laptop.